aws/data_source_aws_vpc_peering_connection.go

```diff
@@ -67,12 +67,12 @@ func dataSourceAwsVpcPeeringConnection() *schema.Resource {
 			"accepter": {
 				Type:     schema.TypeMap,
 				Computed: true,
-				Elem:     schema.TypeBool,
+				Elem:     &schema.Schema{Type: schema.TypeBool},
```

Comment:
Just ran a few tests and one is failing for this particular data source with:

```
$ make testacc TEST=./aws TESTARGS='-run=TestAccDataSourceAwsVpcPeeringConnection_'

==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -run=TestAccDataSourceAwsVpcPeeringConnection_ -timeout 120m
=== RUN   TestAccDataSourceAwsVpcPeeringConnection_basic
--- FAIL: TestAccDataSourceAwsVpcPeeringConnection_basic (17.47s)
	testing.go:503: Step 0 error: Error applying: 6 error(s) occurred:
		
		* data.aws_vpc_peering_connection.test_by_requester_cidr_block: data.aws_vpc_peering_connection.test_by_requester_cidr_block: multiple VPC peering connections matched; use additional constraints to reduce matches to a single VPC peering connection
		* data.aws_vpc_peering_connection.test_by_owner_ids: data.aws_vpc_peering_connection.test_by_owner_ids: multiple VPC peering connections matched; use additional constraints to reduce matches to a single VPC peering connection
		* data.aws_vpc_peering_connection.test_by_accepter_vpc_id: data.aws_vpc_peering_connection.test_by_accepter_vpc_id: accepter.allow_remote_vpc_dns_resolution: '' expected type 'string', got unconvertible type 'bool'
		* data.aws_vpc_peering_connection.test_by_id: data.aws_vpc_peering_connection.test_by_id: accepter.allow_vpc_to_remote_classic_link: '' expected type 'string', got unconvertible type 'bool'
		* data.aws_vpc_peering_connection.test_by_requester_vpc_id: data.aws_vpc_peering_connection.test_by_requester_vpc_id: accepter.allow_vpc_to_remote_classic_link: '' expected type 'string', got unconvertible type 'bool'
		* data.aws_vpc_peering_connection.test_by_accepter_cidr_block: data.aws_vpc_peering_connection.test_by_accepter_cidr_block: multiple VPC peering connections matched; use additional constraints to reduce matches to a single VPC peering connection
	testing.go:563: Error destroying resource! WARNING: Dangling resources
		may exist. The full state and error is shown below.
		
		Error: Error refreshing: 1 error(s) occurred:
		
		* data.aws_vpc_peering_connection.test_by_id: 1 error(s) occurred:
		
		* data.aws_vpc_peering_connection.test_by_id: data.aws_vpc_peering_connection.test_by_id: accepter.allow_classic_link_to_remote_vpc: '' expected type 'string', got unconvertible type 'bool'
		
		State: <nil>
```

@jimmy-btn Could you investigate here? Thanks!

Reply:
Sorry; I forgot to update this.

I have confirmed the issue and am working on a fix to Terraform to address.